bitbots_bringup/launch/simulator_teamplayer.launch

```diff
@@ -26,27 +26,19 @@
         <arg name="basler" default="$(arg basler)"/>
     </include>
 
-    <!-- launch motion and gazebo with soccer field -->
+    <!-- launch motion and webots with soccer field -->
     <group if="$(arg use_fake_walk)">
-        <include file="$(find gazebo_ros)/launch/empty_world.launch">
-            <!-- use simpler physics when faking walk -->
-            <arg name="world_name" value="$(find humanoid_league_gazebo_world)/worlds/robocup_soccer_simple_physics.world"/>
-            <arg name="paused" value="false"/>
+        <include file="$(find wolfgang_webots_sim)/launch/simulation.launch">
```

Comment:
How does the fake walk work in webots? In both cases the same launch file is included.

Reply:
True, there is no fake walk yet. I opened an issue for this. I think the PR can be merged like this and then we will do a new PR adding the fake walk later